server/src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -800,7 +819,7 @@ public void readFrom(StreamInput in) throws IOException {
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         waitForActiveShards.writeTo(out);
-        out.writeString(type);
+        out.writeString(type());
```

Comment:
Same here, would be nice to transfer the comment from the other writeTo() method.